uwsgi.ini

```diff
@@ -1,12 +1,17 @@
 [uwsgi]
+; Set up uwsgi HTTP router on the specified address with HTTP
 http = :5000
+http-keepalive=1
```

Comment:
We should add a comment that this enables `http-keepalive`, rather than sets a duration of 1.

Reply:
I don't think that's correct, it looks like it *is* setting the duration https://github.com/unbit/uwsgi/blob/uwsgi-2.0/plugins/http/http.c#L658

Reply:
🤔  That line you've linked to implies it only does something when it is greater than 1..

The assumption that it was a boolean came from reading discussions like this - https://github.com/unbit/uwsgi/issues/2018.

If it is the timeout in seconds, it should use the `HTTP_KEEP_ALIVE` environment variable to be consistent with the Gunicorn implementation.